index.js

```diff
@@ -1,26 +1,120 @@
 // 1.Fibonacci
 
-// function fibo(n) {
-//   return n <= 1 ? n : fibo(n - 1) + fibo(n - 2);
-// }
-// fibo(5);
-// fibo(6);
-// fibo(7);
+function fibo(n) {
+  return n <= 1 ? n : fibo(n - 1) + fibo(n - 2);
+}
+fibo(5);
+fibo(6);
+fibo(7);
 
 // 2. Implementing with reduce
 
 //concat;
+let arr = [1, 2];
```

Comment:
Please write reusable functions, as concatFn, so you are able to reuse this functionality